src/integrations/admin/import-integration.php

```diff
@@ -94,15 +95,18 @@ public function enqueue_import_script() {
 		\wp_enqueue_style( 'dashicons' );
 		$this->asset_manager->enqueue_script( 'import' );
 
+		$import_failure_alert = $this->get_import_failure_alert();
+
 		$data = [
 			'restApi' => [
 				'root'                => \esc_url_raw( \rest_url() ),
 				'importing_endpoints' => $this->get_importing_endpoints(),
 				'nonce'               => \wp_create_nonce( 'wp_rest' ),
 			],
 			'assets'  => [
-				'loading_msg' => \esc_html__( 'The import can take a long time depending on your site\'s size', 'wordpress-seo' ),
-				'spinner'     => \admin_url( 'images/loading.gif' ),
+				'loading_msg'    => \esc_html__( 'The import can take a long time depending on your site\'s size', 'wordpress-seo' ),
+				'import_failure' => '<div class="yoast-measure yoast-import-failure">' . $import_failure_alert . '</div>',
```

Comment:
I think it'd be better to let the js on the frontend determine what the container html should look like and let the php only provide the (html) message.
```suggestion
				'import_failure' => $import_failure_alert,
```